.github/workflows/pypi-publish.yml

```
@@ -26,7 +26,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel babel
+          pip install setuptools wheel "babel<=2.9.1"
```

```suggestion
          pip install setuptools wheel "babel==2.9.1"
```
I would go ahead with this since it is a quick fix, also if possible maybe its worth adding a comment with the issue link?